### PR TITLE
Update TC 24, 31,37 and edit login step

### PR DESCRIPTION
**Pull request by oksanapishchykepam**

### CODE REVIEW

File: src/main/resources/story/web/reminder/MessageWithLinkToPSAppointmentSchedulerMagicButton.story [hunk 1]
```diff
@@ -120,9 +120,10 @@ Then field located `xpath(//*[contains(text(), 'Provider')]/following-sibling::p
 Scenario: Click the "Send appointment request" button and verify the patient receives the message (Step 13)
 Meta:
     @severity 1
-When I click on element located `xpath(//*[contains(text(), 'Submit Request')])`
-When I wait until element located `xpath(//*[contains(text(), 'Pending Appointment')])` appears
-Then the text 'Got it! We'll let you know when your appointment is confirmed' exists
+When I click on element located `xpath(//*[contains(text(), 'Review and book')]/parent::div/div/descendant::*[contains(text(), 'SEND APPOINTMENT REQUEST') or contains(text(), 'Submit Request')])`
```
Comment (NatalliaSamarava) [inline src/main/resources/story/web/reminder/MessageWithLinkToPSAppointmentSchedulerMagicButton.story:123]: Same as above

**Comment by oksanapishchykepam** [comment]
[allure_MessageWithNoLinkDEMO.zip](https://github.com/lumahealthhq/automated-tests/files/8109233/allure_MessageWithNoLinkDEMO.zip)

**Comment by oksanapishchykepam** [comment]
[allure_SendWaitingRoom_DEMO.zip](https://github.com/lumahealthhq/automated-tests/files/8109432/allure_SendWaitingRoom_DEMO.zip)

File: src/main/resources/story/web/customActions/SendWaitingRoomTextMagicButton.story [hunk 2]
```diff
@@ -73,6 +73,7 @@ Given I am on a page with the URL '${baseApplicationUrl}/patients/${patientId}/c
 When I wait until element located `xpath(//*[contains(text(),'Thanks')])` appears
```
Comment (NatalliaSamarava) [inline src/main/resources/story/web/customActions/SendWaitingRoomTextMagicButton.story:73]: replace with ```When I wait until element located `xpath(//*[contains(text(),'Thanks for letting us know')])` appears```

File: src/main/resources/story/web/customActions/SendWaitingRoomTextMagicButton.story [hunk 3]
```diff
@@ -73,6 +73,7 @@ Given I am on a page with the URL '${baseApplicationUrl}/patients/${patientId}/c
...
 Then the text 'Thanks for letting us know, we'll begin the check-in process shortly.' exists
 Given I request messages for patient with id '${patientId}' and access token '${accessToken}'
+When I wait until element located `By.xpath(//div[text()='arrived'])` appears
```
Comment (NatalliaSamarava) [inline src/main/resources/story/web/customActions/SendWaitingRoomTextMagicButton.story:76]: remove this wait

**Review by NatalliaSamarava — CHANGES_REQUESTED** [review]

**Review by NatalliaSamarava — APPROVED** [review]

**Pull request merged**